starter_code/queries.md

```diff
@@ -4,80 +4,99 @@
 
 ### 1. All the companies whose name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+{name: 'Babelgum'}
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
-<!-- Your Code Goes Here -->
+filter: { number_of_employees: { $gte: 5000 } } 
+sort: {number_of_employees: 1}
+limit: 20
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
 
-<!-- Your Code Goes Here -->
+filter: { number_of_employees: { $gte: 5000 } }  
+project: {name:1, _id:0, founded_year:1}  
```

Comment:
I think you forget to copy paste the filter^^
{$and:[{founded_year: {$gte: 2000}},{founded_year: {$lte: 2005}}]}
And there is no limit and sort